flow/packet-headers/ipv4.yaml

```diff
@@ -403,3 +409,19 @@ components:
             default: 0
             features: [count, metric_tags]
           x-field-uid: 6
+    Flow.Ipv4.Auto:
+      description: |-
+        The OTG implementation can provide a system generated, value for this property.
+      type: object
+      required: [choice]
+      properties:
+        choice:
+          description: |- 
+            The method to be used to provide the system generated value.
```

Comment:
maybe `field or method`? dhcp doesn't sound like a method

Reply:
@apratimmukherjee @SuryyaKrJana do you have some suggestions regarding the nomenclature ?

Reply:
no, Actually dhcp is fine